components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -193,7 +193,7 @@ export default class AnnouncementBar extends React.PureComponent<Props, State> {
                                 >
                                     {(linkmessage) => (
                                         <ToggleModalButton
-                                            ariaLabel={linkmessage}
+                                            ariaLabel={linkmessage as unknown as string}
```

Comment:
I couldn't make it work because type of `linkmessage` is inferred as `ReactNodeArray`